backend/src/apiserver/server/job_server_test.go

```diff
@@ -877,6 +878,9 @@ func TestCreateRecurringRun(t *testing.T) {
 		PipelineSource: &apiv2beta1.RecurringRun_PipelineSpec{PipelineSpec: pipelineSpecStruct},
 		RuntimeConfig: &apiv2beta1.RuntimeConfig{
 			PipelineRoot: "model-pipeline-root",
+			Parameters: map[string]*structpb.Value{
+				"text": &structpb.Value{Kind: &structpb.Value_StringValue{StringValue: "world"}},
```

Comment:
nit: this can be simplified as `"text": structpb.NewStringValue("world"),`

same for below

Reply:
Good point! I will change this.